GitUI/CommandsDialogs/FormBrowse.Designer.cs

```diff
@@ -82,7 +82,8 @@ private void InitializeComponent()
             this.revisionGpgInfo1 = new GitUI.CommandsDialogs.RevisionGpgInfoControl();
             this.FilterToolTip = new System.Windows.Forms.ToolTip(this.components);
             this.statusStrip = new System.Windows.Forms.StatusStrip();
-            this.closeStatusBarItem = new System.Windows.Forms.ToolStripStatusLabel();
+            this.tsslblTelemetryEnabled = new System.Windows.Forms.ToolStripStatusLabel();
```

Comment:
I do not want to occupy space to just show that telemetry is enabled/disabled

Reply:
I'm not 100% happy with the status bar myself. And with the latest change that prompts a user to opt-in/out, I've been thinking of moving it to menus - https://github.com/gitextensions/gitextensions/pull/6522#issuecomment-494367466

Reply:
As this is set during installation (or first start) it should be enough, no need to constantly inform the user in the toolbar. (about form?)

Reply:
The About form is kind of overload already, but can add to the info being copied/reported by NBug

Reply:
About would be instead of status bar no need in the report